torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2321,6 +2323,13 @@ def create_symbol(
     ) -> "sympy.Expr":
         assert isinstance(source, Source), f"{type(source)} {source}"
         assert not (positive and val < 0), f"positive set for negative value: {val}"
+
+        if dynamic_dim is DimDynamic.UNBACKED:
+            # TODO: this is naughty
+            s = self.create_unbacked_symint()
+            constrain_range(s, min=2)
```

Comment:
Not directly related to this PR, but should `constrain_range` (and `constrain_unify`) add corresponding assertions to TV?

Reply:
Yeah, they probably should.

Reply:
AFAIU, we don't generate guards on those `constrain_*` calls. So, I'm guessing they should be treated as target expressions in the TV, right?

Reply:
Actually this is trickier than I thought. constrain_range is typically called on unbacked symints. These symints can never have guards, because they only show up midway through execution. So maybe no TV is needed